From e54065494eaa3f3550d40c4e992b30c39bfecfc0 Mon Sep 17 00:00:00 2001
From: annsai01 <annam.saimanisha@arm.com>
Date: Mon, 20 Feb 2023 11:26:23 +0000
Subject: [PATCH] arm: total_compute: Remap console logs

Remapping console logs from soc uart2 (s1 termial) to css non-secure (uart_ap terminal)

Change-Id: I554f3ab6eb8439c54f5568c47e48f1543ac67b44
Signed-off-by: Annam Sai Manisha <annam.saimanisha@arm.com>
---
 include/configs/total_compute.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/total_compute.h b/include/configs/total_compute.h
index 0324b1e1b2..d3e77377f2 100644
--- a/include/configs/total_compute.h
+++ b/include/configs/total_compute.h
@@ -2,7 +2,7 @@
 /*
  * Configuration for Total Compute platform. Parts were derived from other ARM
  * configurations.
- * (C) Copyright 2020 Arm Limited
+ * (C) Copyright 2020-2023 Arm Limited
  * Usama Arif <usama.arif@arm.com>
  */
 
@@ -14,7 +14,8 @@
 
 #define CONFIG_SYS_BOOTM_LEN	(64 << 20)
 
-#define UART0_BASE		0x7ff80000
+/* CSS non-secure UART base address */
+#define UART0_BASE		0x2A400000
 
 /* PL011 Serial Configuration */
 #define CONFIG_PL011_CLOCK	7372800
-- 
2.39.1

